libraries/chain/controller.cpp

```diff
@@ -3464,13 +3456,13 @@ struct controller_impl {
          if (s == controller::block_status::incomplete) {
             const auto& id = chain_head.id();
             const auto& new_b = chain_head.block();
-            br.total_time += fc::time_point::now() - start;
+            fc::time_point now = fc::time_point::now();
 
             ilog("Produced block ${id}... #${n} @ ${t} signed by ${p} "
-                 "[trxs: ${count}, lib: ${lib}${confs}, net: ${net}, cpu: ${cpu}, elapsed: ${et}, time: ${tt}]",
+                 "[trxs: ${count}, lib: ${lib}${confs}, net: ${net}, cpu: ${cpu}, elapsed: ${et} us, time: ${tt} us]",
```

Comment:
Any reason to only add `us` to "elapsed" and "time" but not "cpu"?

Reply:
Thanks for spotting this. There was no reason. I overlooked `cpu`. Will add `us` in a future PR.